server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -707,6 +710,37 @@ async fn resend_webhook(
     Ok((StatusCode::ACCEPTED, Json(EmptyResponse {})))
 }
 
+async fn expunge_attempt_content(
+    State(AppState { ref db, .. }): State<AppState>,
+    Path((_app_id, msg_id, attempt_id)): Path<(
+        ApplicationIdOrUid,
+        MessageIdOrUid,
+        MessageAttemptId,
+    )>,
+    permissions::OrganizationWithApplication { app }: permissions::OrganizationWithApplication,
+) -> Result<StatusCode> {
+    let msg = ctx!(
+        message::Entity::secure_find_by_id_or_uid(app.id, msg_id)
+            .one(db)
+            .await
+    )?
+    .ok_or_else(|| HttpError::not_found(None, None))?;
```

Comment:
We're definitely inconsistent about this, but I think in situations like this it's helpful to add a bit more information to the `not_found(...)` error. As it is now, if a request 404s it's not clear if that's because the app_id was wrong, the msg_id was wrong, or the attempt_id was wrong.

Reply:
I added a message (but not a separate code) to each of the errors to say if it errored on the message or the message attempt.